webpage_info/getting-started/1. Devfile Ecosystem/Create, Share, and Consume Devfiles.adoc

```diff
@@ -0,0 +1,33 @@
+== *Create, Share and Consume Devfiles*
+:imagesdir: ../../../images
+'''''
+The following is an example of a devfile ecosystem that is framed within the context of an organization looking to standardize their development environment.  In the simplest case, developers can just consume the devfiles that are available from the public community registry.  If the organization sees the need to author and share their own devfiles internally, they will need a role based approach where Developers, Devfile Authors, and Registry Administrators can interact together.
+
+image::devfileEcosystem.svg[]
+
+
+=== *The Main Areas of the Devfile Ecosystem*
+=== *Create*
+
+A devfile author aka “stack provider” can be an individual or a group representing a runtime vendor.  This role requires sound knowledge of the supported runtime so devfiles can be created with the proper guidance on how to build and run applications.
+
+If a runtime stack is not available in the public registry, an organization can choose to develop their own and keep it private for their in-house development.
+
+=== *Share*
+
+The public community registry is managed by the community and hosted by Red Hat.  This is one way to share your devfile if there is a benefit to the greater community.
+
+If an organization wants to keep their own devfiles private but wishes to share with other departments, they can assign a Devfile Registry Administrator to deploy, maintain, and manage the contents of their private registry and the default list of devfile registries available in a given cluster.  With this set up, they can use devfiles from both the public and private registries, which gives them the flexibility of options.
+
+=== *Consume*
+
+Devfiles can be made available to developers via supported tools*^*^*.  Many of the existing tools offer a way to register or catalog public/private Devfile Registries which then allows the tool to expose the devfiles for development.
```

Comment:
Put the attention on the user/role, avoid latin phrases like "via, per, etc.", and I'm not sure what `*^*^*' means? 

`Developers use the supported tools to access devfiles.` 

For the second sentence, the IBM SG wants us to avoid `/` so simply say, `...a way to register or catalog public and private devfile registries...`

Reply:
I was trying to  bold and superscript the asterisk :smile: .   I updated it to remove the bold effect since it wasn't rendering.  It should look like this: 
<img width="471" alt="image" src="https://user-images.githubusercontent.com/84398375/166325672-c949d11a-ac3d-4fa5-b317-ef2a96a3ba1b.png">
with an explanation at the bottom of the section: 
<img width="940" alt="image" src="https://user-images.githubusercontent.com/84398375/166325744-8acd06f1-14b9-4e22-8ea4-61393177f5a5.png">